docs/concat_merge_usage.rst

```diff
@@ -0,0 +1,52 @@
+Usage scenarios
```

Comment:
I think these could be in the `cmdline.rst` file – they'd probably be easier to find there.